corehq/apps/reports/display.py

```diff
@@ -72,17 +72,10 @@ def readable_form_name(self):
 
 class _FormType(object):
 
-    def __init__(self, domain, xmlns, app_id=None):
+    def __init__(self, domain, xmlns, app_id):
```

Comment:
Is this just always called with `app_id`?

It does seem like strange logic though since the `get_form_analytics_metadata` function uses `app_id`

Reply:
Yeah, there's only one instance where this is being used and it passes the app_id.